src/gc/sample/gchandletable_forsample.cpp

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// 
+
+// This file just includes the regular gchandletable.cpp file, but adds some
+// headers that are necessary for it to build in the GCSample project.
+
+#include "common.h"
+#include "gcenv.h"
+
```

Comment:
You should be able to `#include "gcenv.h"` in gchandletable.cpp, so that you do not need this separate files. gcenv.h is included in all other GC files.